src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -5,6 +5,9 @@
  * Do not remove no-args constructor
  */
 public class Excavator extends Machine {
+    private int sizeBucket;
```

Comment:
Don't forget about access modifiers for your custom fields. It's a good practice to make fields private and provide getters and setters for accessing them.